lib/rbac/filterer.rb

```diff
@@ -54,6 +54,16 @@ class Filterer
 
     TAGGABLE_FILTER_CLASSES = CLASSES_THAT_PARTICIPATE_IN_RBAC - %w(EmsFolder) + %w(MiqGroup User)
 
+    NETWORK_MODELS_FOR_BELONGSTO_FILTER = %w(
+      CloudNetwork
+      CloudSubnet
+      NetworkRouter
+      SecurityGroup
+      FloatingIp
+      NetworkPort
+      LoadBalancer
```

Comment:
For the future, we should find a way to not hardcode this list because we'll need to update this list the moment we add something new.

Reply:
Can this be sorted alphabetically?

Reply:
@jrafanie 👍 absolutely agree,  @bdunne 👍